integreat_cms/sitemap/sitemaps.py

```diff
@@ -182,7 +186,7 @@ def __init__(self, region, language):
         # Filter queryset based on region and language
         self.queryset = self.queryset.filter(
             event__in=self.region.events.all(), language=self.language
-        )
+        ).order_by("-version")
```

Comment:
```suggestion
        ).distinct("event__pk")
```
This should be sufficient (same for pois)